src/app.js

```diff
@@ -25,74 +53,38 @@ export const start = async args => {
 
   const username = program.user;
 
-  let reposList;
-
-  //TODO: prompt for username if none provided
   if (!username) {
-    console.error(
-      `Use option -u or --user to search repositories by Github use
+    process.stderr.write(
+      `Use option -u or --user to search repositories by Github user
       \nUsage: git-started -u jwu910`.trim(),
     );
 
     return;
   }
 
   try {
-    const response = await getRepos(username);
-
-    const repositories = response.user.repositories.edges;
-
-    if (!repositories.length) {
-      throw new Error('No repositories found.');
-    }
-
-    reposList = repositories.map(({ node }) => {
-      const { id, name, stargazers } = node;
-
-      let { description } = node;
-
-      const termWidth = process.stdout.columns;
-
-      if (description === null) {
-        description = 'No Description';
-      }
-
-      const limit = termWidth - name.length;
-
-      return {
-        title: `${name} - ${stargazers.totalCount} stars - ${truncate(
-          description,
-          limit - 20,
-          '...',
-        )}`,
-        value: {
-          id,
-          repo: name,
-          owner: username,
-        },
-      };
-    });
+    const userGraph = await getRepos(username);
+    const reposList = reposListFromGraphEdges(
```

Comment:
See my comment above about the reposListFromGraphEdges abstraction.

Note about the line spacing here too. I would like to keep like declarations together, but line break between declarations that are dependent on the previous line. 

example:

```
const myFoo = 'foo';
const myBar = 'bar';

// and
const myFoo = 'foo';

const myBar = someFunc(myFoo);
```